server/routes/induction/create/reasonsNotToGetWorkCreateController.test.ts

```diff
@@ -0,0 +1,202 @@
+import { Request, Response } from 'express'
+import type { SessionData } from 'express-session'
+import type { InductionDto } from 'inductionDto'
+import type { ReasonsNotToGetWorkForm } from 'inductionForms'
+import aValidPrisonerSummary from '../../../testsupport/prisonerSummaryTestDataBuilder'
+import ReasonsNotToGetWorkCreateController from './reasonsNotToGetWorkCreateController'
+import ReasonNotToGetWorkValue from '../../../enums/reasonNotToGetWorkValue'
+import QualificationLevelValue from '../../../enums/qualificationLevelValue'
+
+describe('reasonsNotToGetWorkCreateController', () => {
+  const controller = new ReasonsNotToGetWorkCreateController()
+
+  const prisonNumber = 'A1234BC'
+
+  let req: Request
+  let res: Response
+  const next = jest.fn()
+
+  const noErrors: Array<Record<string, string>> = []
+
+  beforeEach(() => {
+    jest.resetAllMocks()
+    req = {
+      session: {} as SessionData,
+      body: {},
+      user: {} as Express.User,
+      params: { prisonNumber } as Record<string, string>,
+      flash: jest.fn(),
+    } as unknown as Request
+
+    res = {
+      redirect: jest.fn(),
+      render: jest.fn(),
+    } as unknown as Response
+  })
+
+  describe('getReasonsNotToGetWorkView', () => {
+    it('should get the reasons not to get work create view', async () => {
+      // Given
+      const prisonerSummary = aValidPrisonerSummary()
+      req.session.prisonerSummary = prisonerSummary
+
+      const inductionDto: InductionDto = undefined
```

Comment:
I think you'll always have an induction on the session, because the previous screen (Hoping to get work) will have set it up (albeit with a small handful of properties set)

This should do it I think:
```
const inductionDto: InductionDto = {
  workOnRelease: {
    hopingToWork: HopingToGetWorkValue.NO,
  },
}
req.session.inductionDto = inductionDto
```